pkg/controller/util.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Would a better name for this file be "constants.go" maybe? `util.go` is pretty generic, and for now really all it's doing is holding constants.

Reply:
True. I can change this.